lib/Gateway.js

```diff
@@ -567,6 +570,26 @@ function getNodeName (node, ignoreLoc) {
   )
 }
 
+/**
+ *  Return re-arranged based on critical CCs
+ *
+ * @param {node.valueId} values valueId keys
+ * @returns An nested array, first value is priority CC values and second the remaining
+ */
+
+function getPriorityCCFirst (values) {
+  const priorityList = []
+  const remainingList = []
+  for (const k in values) {
+    if (Constants.priorityCommandClass.includes(values[k].commandClass)) {
+      priorityList.push(k)
```

Comment:
You can use a single array and use the unshift command to push in the front of the array

Reply:
good one, done that!